one.mixin.messenger.yml

```diff
@@ -0,0 +1,45 @@
+id: one.mixin.messenger
+runtime: org.gnome.Platform
+runtime-version: '42'
```

Comment:
so in 6 month when runtime 42 is EOL, what's the plan?

Reply:
We could add a libgtkwebkit module then build the shared library for ourself, It might need times to do research and test.

Reply:
Have you tested changing it to 43 now? There's rarely a good reason to use the older runtime when a newer one is available.

Reply:
> Have you tested changing it to 43 now?

Yes, I tried switching to GNOME-43. 

but I got an runtime error when I try to run:
```
mixin_desktop: error while loading shared libraries: libwebkit2gtk-4.0.so.37: cannot open shared object file: No such file or directory
```